Yandex/YandexAdapter/ALYandexMediationAdapter.m

```diff
@@ -408,20 +429,24 @@ - (void)interstitialAdDidFailToPresent:(YMAInterstitialAd *)interstitial error:(
     [self.delegate didFailToDisplayInterstitialAdWithError: adapterError];
 }
 
-- (void)interstitialAd:(YMAInterstitialAd *)interstitialAd willPresentScreen:(nullable UIViewController *)webBrowser
+- (void)interstitialAdDidClick:(YMAInterstitialAd *)interstitialAd
+{
+    [self.parentAdapter log: @"Interstitial ad clicked"];
+    [self.delegate didClickInterstitialAd];
+}
+
+- (void)interstitialAd:(YMAInterstitialAd *)interstitialAd willPresentScreen:(UIViewController *)webBrowser
 {
     // This callback and interstitialWillLeaveApplication are mutually exclusive
     // Ad either opens in-app or redirects to another app
```

Comment:
@Billy-Hu - Let's also remove these comments from the adapter since they're no longer applicable.